Index.html

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="style.css">
+    <!-- To embed a font -->
+    <style>
+        @import url('https://fonts.googleapis.com/css2?family=Inter:wght@400;600;700&display=swap');
+    </style>
+    <!-- Font 'Inter' embeded-->
+</head>
+<body>
+    <header class="toolbar">
+        <span class="welcome">Welcome</span>
+        <span class="hamburguer"><img src="./Images/Union.png" alt="hamburguer menu"></span>
+        
+    </header>
+    <section class="headline">
+        <h1>
+            Hey there. I'm Cindy
+            <br>
+            <span class="greenheader">I'm a software developer</span>
+        </h1>
```

Comment:
- [ ] Kindly make the title in an `h1` tag and the subtitle in an `hx` tag (examples: `<h1>,<h2>,<h3>,` etc...).
- [ ] so for example:-
```
<h1>Hey there. I'm Cindy</h1>
<h2>I'm a software developer</h2>
```